opm/simulators/timestepping/AdaptiveTimeStepping.hpp

```diff
@@ -181,10 +181,10 @@ void registerAdaptiveParameters();
                              Solver& solver,
                              const bool isEvent,
                              const std::vector<int>* fipnum = nullptr,
-                             const std::function<bool()> tuningUpdater = [](){return false;})
+                             const std::function<bool(const double, const double, const int)> tuningUpdater = [](){return false;})
```

Comment:
it is surprising indeed, but it seems std::function doesn't actually check the lambda provided and this would just crash on runtime. since the default value is never used it was not really a problem per se. 

i was supposed to do a precursor PR but I never got around to it. I have opened that now, see https://github.com/OPM/opm-simulators/pull/5803